docs/tutorials/multimodal/beginner_multimodal.md

```diff
@@ -15,14 +15,14 @@ np.random.seed(123)
 
 ## Dataset
 
-For demonstration, we use the [PetFinder dataset](https://www.kaggle.com/c/petfinder-adoption-prediction). The PetFinder dataset provides information about shelter animals that appear on their adoption profile to predict the animals' adoption rates, grouped into five categories, hence a multi-class classification problem.
+For demonstration, we use a simplified and subsampled version of [PetFinder dataset](https://www.kaggle.com/c/petfinder-adoption-prediction), which provides information about shelter animals that appear on their adoption profile to predict the animals' adoption rates, grouped into two categories, hence a binary classification problem.
```

Comment:
Can you add one more sentence here to clarify what are the two categories? That should be helpful for beginners.